unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Camera/CameraController.cs

```diff
@@ -80,6 +81,7 @@ private void PrecisionChanged(Vector3 newValue, Vector3 oldValue)
 
     private void Update()
     {
+        Transform cameraTransform = this.camera.transform;
```

Comment:
Can't we do this assignement only one time in the Start or Awake event and store it in a private variable?

Reply:
Indeed. Let me try.

Reply:
Done